Sources/Kommunicate/Classes/FaqViewController.swift

```diff
@@ -18,6 +18,7 @@ public class FaqViewController: UIViewController, Localizable {
         self.url = url
         self.configuration = configuration
         super.init(nibName: nil, bundle: nil)
+        ALKCustomEventHandler.shared.publish(triggeredEvent: CustomEvent.faqClick, data:["UserSelection":["FaqUrl":url]] )
```

Comment:
Add a KM typealias for `ALKCustomEventHandler`.

Reply:
@mukeshthawani added